setup.py

```diff
@@ -273,11 +271,11 @@ def get_long_description():
             Operating System :: POSIX :: Linux
             Programming Language :: Python
             Programming Language :: Python :: 2
```

Comment:
Presumably should remove this line as well.

Reply:
Thanks. Removed.